bundledata.go

```diff
@@ -571,6 +634,24 @@ func (verifier *bundleDataVerifier) verifyPlacement(to []string) {
 	}
 }
 
+func (verifier *bundleDataVerifier) verifyKubernetesPlacement(name, placement string, to []string) {
+	if len(to) > 1 || len(to) == 1 && placement != "" && placement != to[0] {
+		if placement != "" {
+			verifier.addErrorf("inconsistent placement (%s) and to (%s) for application %q", placement, to, name)
```

Comment:
This error is subtly different. The other error you might be thinking of is for when the count differs. Here, if they have specified both and there's only one to option, that values need to match. No one would ever do that but just in case, we enforce consistency.